Storage/Storage.xcodeproj/project.pbxproj

```diff
@@ -47,7 +47,7 @@
 		262B270E2621412000A421CF /* ProductAddOn+CoreDataProperties.swift in Sources */ = {isa = PBXBuildFile; fileRef = 262B26752621401A00A421CF /* ProductAddOn+CoreDataProperties.swift */; };
 		262B27132621412700A421CF /* ProductAddOnOption+CoreDataClass.swift in Sources */ = {isa = PBXBuildFile; fileRef = 262B26762621401A00A421CF /* ProductAddOnOption+CoreDataClass.swift */; };
 		262B27182621412D00A421CF /* ProductAddOnOption+CoreDataProperties.swift in Sources */ = {isa = PBXBuildFile; fileRef = 262B26772621401A00A421CF /* ProductAddOnOption+CoreDataProperties.swift */; };
-		263E37D72641AD5100260D3B /* BuildFile in Frameworks */ = {isa = PBXBuildFile; productRef = 263E37D62641AD5100260D3B /* SwiftPackageProductDependency */; };
+		263E37D72641AD5100260D3B /* Codegen in Frameworks */ = {isa = PBXBuildFile; productRef = 263E37D62641AD5100260D3B /* Codegen */; };
```

Comment:
Do you know what could be triggering these changes?

Reply:
Hmm, not sure. I added `GeneratedCopiable` protocol to `Storage.EligibilityErrorInfo`, and ran `rake generate`.. I'll double check to ensure. 👍

Reply:
Yep, this is because of the Codegen. Tested this on a fresh branch.

Created this in `Storage` and ran `rake generate`:
```swift
import Codegen

struct TestModel: Codable, Equatable, GeneratedCopiable {
    let id: String
    let name: [String]
}
```

I'm getting the same changes:
```diff
...
-               263E37D72641AD5100260D3B /* BuildFile in Frameworks */ = {isa = PBXBuildFile; productRef = 263E37D62641AD5100260D3B /* SwiftPackageProductDependency */; };
+               263E37D72641AD5100260D3B /* Codegen in Frameworks */ = {isa = PBXBuildFile; productRef = 263E37D62641AD5100260D3B /* Codegen */; };
...
```
```diff
...
-               57150E1324F462DF00E81611 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; productRef = 57150E1224F462DF00E81611 /* SwiftPackageProductDependency */; };
+               57150E1324F462DF00E81611 /* TestKit in Frameworks */ = {isa = PBXBuildFile; productRef = 57150E1224F462DF00E81611 /* TestKit */; };
...